xml/System.Diagnostics.CodeAnalysis/UnscopedRefAttribute.xml

```diff
@@ -20,8 +20,18 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Used to indicate a byref escapes and is not scoped.</summary>
+    <remarks>
+      <para> There are several cases where the C# compiler treats a <see langword="ref" /> as implicitly <see langword="scoped" /> - where the compiler does not allow the <see langword="ref" /> to escape the method.</para>
+      <para> For example:</para>
+      <li>
```

Comment:
I changed this on purpose: I want to see if the preview respects `<li>` and `<ol>`, unlike `<list>` and `<item>`, which it doesn't like.

Reply:
Nope, didn't like it. The preview behaved a little bit better than with list+item, but still doesn't show the bullets correctly:

![image](https://user-images.githubusercontent.com/1175054/191638589-4c346f82-e2b8-4ee6-8597-a3da7a78426f.png)

Reply:
Well duh, it's because I used li and ol incorrectly. Fixed. Let's see if it works this time.

Reply:
It worked!

![Screenshot_20220921-203424_Edge](https://user-images.githubusercontent.com/1175054/191652744-9194e843-33e1-4980-8dff-d80caf428147.jpg)